test/puppetlabs/kitchensink/core_test.clj

```diff
@@ -108,6 +108,17 @@
        ["foo" "bar" "baz"] "foo, bar, and baz"
        ["foo" "bar" "baz" "qux"] "foo, bar, baz, and qux"))
 
+(deftest key->str-test
+  (testing "returns strings unmodified"
+    (is (= "foo" (key->str "foo")))
+    (is (= ":foo" (key->str ":foo")))
+    (is (= "foo/bar" (key->str "foo/bar"))))
+
+  (testing "returns the entire stringified keyword"
```

Comment:
It might be helpful to encode the expected fifference between this and `name` around namespaces. So maybe a test that asserts `(keyword->str ::foo)` preserves the namespace in the result?

Reply:
Is that not what the last case tests? `(key->str :foo/bar)`

Reply:
ah, yeah it does. I missed that case, i was thinking more of the other side (where a namespace would be "added").